v2/internal/pkg/operator/common.go

```diff
@@ -170,7 +182,18 @@ func (o OperatorCollector) prepareD2MCopyBatch(images map[string][]v2alpha1.Rela
 				} else {
 					src = src + ":" + imgSpec.Digest
 				}
-				dest = dest + ":" + imgSpec.Digest
+				if o.generateV1DestTags {
+					hasher := fnv.New32a()
```

Comment:
@sherine-k - just out of curiosity why did you use Fowler-Noll-Vo hash ?

Reply:
I'm just trying to get the tag to be identical to what v1 generates.
v1 relies on [oc adm catalog mirror](https://github.com/openshift/oc/blob/bdce85daf64eda40dd1b604270b85c67a296bc93/pkg/cli/admin/catalog/mirrorer.go#L176-L185) to create a tag when no tag is provided.